### PR TITLE
Introduce parallel openstack Metal3 image building

### PR DESCRIPTION
**Pull request by smoshiur1237**
This PR will add the parallel openstack Metal3 image building process for ubuntu and centos. 
Fixes #561

### CODE REVIEW

File: ci/jobs/openstack_image_building.pipeline [hunk 1]
```diff
@@ -106,54 +102,59 @@ pipeline {
...
-              sh "docker run --rm \
-                ${DOCKER_CMD_ENV}\
-                -v ${CURRENT_DIR}:/data \
-                -v ${METAL3_CI_USER_KEY}:/data/id_ed25519_metal3ci \
-                registry.nordix.org/metal3/image-builder \
-                /data/ci/images/gen_metal3_ubuntu_image.sh \
-                /data/id_ed25519_metal3ci 1"
-
+    stage('Building Metal3 image for Karlskrona region'){
+      parallel {
```
[the remark this answers is not in the document]
Comment (smoshiur1237) [reply ci/jobs/openstack_image_building.pipeline:111]: I think it is possible to run parallel run for both region. But the discussion I had earlier with others, lead towards the decision that I need to make parallel run for ubuntu and centos image for metal3 or nodes. That is why I am moving this way.

[the remark this answers is not in the document]
Comment (lentzi90) [reply ci/jobs/openstack_image_building.pipeline:111]: Ok let's start with this and then we can see later if we want to parallellize more

**Review by lentzi90 — APPROVED** [review]

File: ci/jobs/openstack_image_building.pipeline (hunk 1 above)
[the remark this answers is not in the document]
Comment (kashifest) [reply ci/jobs/openstack_image_building.pipeline:111]: We can do this also @smoshiur1237 , it didnt come to my mind. You can tackle this here or in a new PR whichever is easier for you.

[the remark this answers is not in the document]
Comment (smoshiur1237) [reply ci/jobs/openstack_image_building.pipeline:111]: It would be better for me to deal with it in separate PR.

**Pull request merged**